client/src/components/projects/project-filter/ProjectFilter.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <div class="filter">
+    <Accordeon
+          name="Production line"
+          :opened="lineAccordionOpened"
+          @update:opened="lineAccordionOpened=!lineAccordionOpened"
+        >
+          <div
+            v-for="line in lines"
+            :key="line.name"
+          >
+          <div class="filter-item">
+              <Checkbox :checked = "line.checked"
+                        v-on:update:checked="handleFilterAction(line, 'line')"/> 
+              <span class="title is-5 is-size-16 is-uppercase">{{ line.name }}</span>
+          </div>
+        </div>
+    </Accordeon>
+    <Accordeon
+          name="Program"
+          :opened="programAccordionOpened"
+          @update:opened="programAccordionOpened=!programAccordionOpened"
+        >
+          <div
+            v-for="program in programs"
+            :key="program.name"
+          >
+            <div class="filter-item">
+              <Checkbox :checked = "program.checked"
+                        v-on:update:checked="handleFilterAction(program, 'program')"/> 
+              <span class="title is-5 is-size-16">{{ program.name }}</span>
+            </div>
+          </div>
+    </Accordeon>
+    <Accordeon
+          name="Project type"
+          :opened="typeAccordionOpened"
+          @update:opened="typeAccordionOpened=!typeAccordionOpened"
+        >
+          <div
+            v-for="t in types"
+            :key="t.name"
+          >
+            <div class="filter-item">
+              <Checkbox :checked = "t.checked"
+                        v-on:update:checked="handleFilterAction(t, 'type')"/>
+              <span class="title is-5 is-size-16">{{ t.name }}</span>
+            </div>
+          </div>
+    </Accordeon>
+    <Accordeon
+          name="Domains"
+          :opened="domainAccordionOpened"
+          @update:opened="domainAccordionOpened=!domainAccordionOpened"
+        >
+          <div
+            v-for="domain in domains"
+            :key="domain.name"
+          >
+            <div class="filter-item">
+              <Checkbox :checked = "domain.checked"
+                        v-on:update:checked="handleFilterAction(domain, 'domain')"/>
+              <span class="title is-5 is-size-16">{{ domain.name }}</span>
+            </div>
+          </div>
+    </Accordeon>
+    <Accordeon
+          v-if="customers && customers.length"
+          name="Customers"
+          :opened="customerAccordionOpened"
+          @update:opened="customerAccordionOpened=!customerAccordionOpened"
+        >
+          <div
+            v-for="customer in convertedCustomers"
+            :key="customer.name"
+          >
+            <div class="filter-item">
+              <Checkbox :checked = "customer.checked"
+                        v-on:update:checked="handleFilterAction(customer, 'customer')" /> 
+              <span class="title is-5 is-size-16">{{ customer.name }}</span>
+            </div>
+          </div>
+    </Accordeon>
```

Comment:
Shoul be refactored to be a map with v-for instead of Accrodion copy paste 5 times